# Potenciales errores
# print(""
# lista = [1,2]
# lista[45]

# Excepciones que podemos capturar
try:
    n1 = int(input("Ingrese un numero: "))
except:
    print("Error, no ingresaste un numero")
